scripts/docker/docker-node.sh

```diff
@@ -47,21 +51,41 @@ if [ "$port_base" -gt 59900 ]; then
   exit 1
 fi
 
+if [ -n "$(docker ps -q -a -f name=^harmony-$account_id-$port_base$)" ]; then
+  echo "Stop node for account id: $account_id (port $port_base)"
+  docker rm -v -f harmony-$account_id-$port_base >/dev/null
```

Comment:
will this remove the downloaded image?  shall we just do docker stop?

Reply:
It will not remove image. Only remove the container. Volumes are not removed. Container must have different names. So need to remove before starting new container.